test/integration/server-selection/server_selection.test.ts

```diff
@@ -0,0 +1,13 @@
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+describe('Server Selection Unified Tests (Node Driver)', function () {
+  /* TODO(NODE-2471) ping on connect
+   * TODO(NODE-5774) duplicate server selection for bulkWrite and other wrapper operations
```

Comment:
I don't think we should be duplicating the tests in this PR - I'd skip them in the regular suite only and/or fix them in the regular spec tests and open a drivers ticket to resolve in the specs repo. This duplication here even with the TODO could be confusing.

Reply:
The node behavior currently performs extra server selections for certain operations (including bulkWrite, the spec test case), since this is node specific behavior, we don't want to open a drivers ticket. Similar duplication exists for SDAM spec v driver tests ([see here](https://github.com/mongodb/node-mongodb-native/tree/main/test/integration/server-discovery-and-monitoring/unified-sdam-node-specs)).

Reply:
I didn't see a precedent was set for this situation, which I disagree with but am happy to concede if the rest of the team is ok with it. My opinion here is 2 concerns:

1. By duplicating the unified tests and modifying them to account for the extra ping, we now have a situation where the unified tests get updated, and we need to remember to sync them in two places and potentially modify them in one. I feel this is creating additional work for us in the future which I'd like to avoid.
2. We already have addressed this issue in the unified runner, [see here](https://github.com/mongodb/node-mongodb-native/blob/main/test/tools/unified-spec-runner/entities.ts#L225), so I'm curious as to why this is happening in the first place, and why we aren't addressing it as part of this ticket?

Reply:
1. I can ask the team for more input. 

2. To my knowledge the `skipPingOnConnect` symbol doesn't stop server selection, but skips sending the ping to the server, is this the desired behavior. Here's the [code snippet](https://github.com/mongodb/node-mongodb-native/blob/b93d405275c3a8ce6b1eba0af335ffb2a309e34e/src/sdam/topology.ts#L455).

Reply:
Yeah I think we should discuss in tech chat or virtual office. It might be an easier approach then to update the matchers in the unified runner itself to ignore the pings when asserting logging spec events, since it currently doesn't look for them.